website/docs/cloud-docs/vcs/bitbucket-data-center.mdx

```diff
@@ -0,0 +1,124 @@
+---
+page_title: Bitbucket Data Center - VCS Providers - Terraform Cloud
+description: >-
+  Learn how to use Bitbucket Data Center for VCS features.
+---
+
+# Configuring Bitbucket Data Center Access
+
+This topic describes how to connect Bitbucket Data Center to Terraform Cloud. For instructions on how to connect Bitbucket Cloud, refer to [Configuring Bitbucket Cloud Access](/terraform/cloud-docs/vcs/bitbucket-cloud). Refer to [Connecting VCS Providers to Terraform Cloud](/terraform/cloud-docs/vcs) for other supported VCS providers.
+
+**Bitbucket Server is deprecated**.  Atlassian ended support for Bitbucket Server on February 15, 2024, and recommends using either Bitbucket Data Center or Bitbucket Cloud instead. Refer to the [Atlassian documentation](https://bitbucket.org/blog/cloud-migration-benefits) for additional information.
+
+Terraform Cloud will end support Bitbucket Server on August 15, 2024. Terraform Enterprise will also end support for Bitbucket Server in Terraform Enterprise v202410. [Contact HashiCorp support](https://support.hashicorp.com/hc/en-us) if you have any questions regarding this change.
+
+
+
+## Overview
+
+The following steps provide an overview of how to connect Terraform Cloud and Terraform Enterprise to Bitbucket Data Center:
+
+1. Add a new VCS provider to Terraform Cloud or Enterprise.
+1. Create a new application link in Bitbucket.
+1. Create an SSH key pair. SSH key must have an empty passphrase because Terraform Cloud cannot use SSH keys that require a passphrase.
+1. Add an SSH key to Bitbucket. You must complete this step as a non-administrator user in Bitbucket.
+1. Add the private SSH key to Terraform. 
+
+## Requirements
+
+- You must have permission to manage VCS settings for the organization. Refer to [Permissions](/terraform/cloud-docs/users-teams-organizations/permissions) for additional information. [permissions-citation]: #intentionally-unused---keep-for-maintainers
+
+- You must have OAuth authentication credentials and an SSH key. Refer to []().
```

Comment:
```suggestion
- You must have OAuth authentication credentials for Bitbucket Data Center. 
```
Is this accurate? I wasn't sure how the original text fit into this. I had left a space to link to relevant instructions, which was breaking Vercel, but I wasn't able to find anything appropriate in the Atlassian docs.

Reply:
Its implied as well as accurate.